internal/utils/browser.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"github.com/pkg/browser"
+)
+
+// BrowserInterface provides access to browser
+type BrowserInterface interface {
```

Comment:
[question] what was the motivation for placing this in a `utils` package and as its own interface?

I'm wondering if it makes sense to somehow merge in the `OpenBrowser(url string)` method onto our existing `terminal.UI` construct?